xclim/indices/fwi.py

```diff
@@ -18,8 +18,8 @@
 Fire season
 -----------
 Fire weather indexes are iteratively computed, each day's value depending on the previous day indexes.
-Additionally and optionally, the codes are "shut down" (set to NaN) in winter. There are a few ways of computing this
-shut down and the subsequent spring start up. The `fire_season` function allows for full control of that,
+Additionally and optionally, the codes are "shut-down" (set to NaN) in winter. There are a few ways of computing this
+shut-down and the subsequent spring start-up. The `fire_season` function allows for full control of that,
```

Comment:
```suggestion
Additionally and optionally, the codes are "shut down" (set to NaN) in winter. There are a few ways of computing this
shut down and the subsequent spring start-up. The `fire_season` function allows for full control of that,
```
"to shut down", as in the verb. https://www.merriam-webster.com/dictionary/shut%20down
You're right for "start-up" though.